bindings/python/tokenizers/implementations/base_tokenizer.py

```diff
@@ -13,6 +13,14 @@ def __repr__(self):
             self._tokenizer.get_vocab_size(),
             ', '.join(k + '=' + str(v) for k, v in self._parameters.items()))
 
+    def num_added_tokens(self, is_pair: bool) -> int:
```

Comment:
Same comment here. I find this really confusing.

Reply:
As the `_tokenizer` property is "kinda" private on `BaseTokenizer`, it's cleaner to forward the method on `BaseTokenizer` and not expose the `_tokenizer` property. 

Its exactly the same behavior as for `get_vocab_size`